### PR TITLE
Suppress swift-testing output when debugging XCTests

### PR DESCRIPTION
**Pull request by plemarquand**
When debugging an XCTest in a project that contains both XCTests and swift-testing tests the `xctest` utility runs both test executables. This results in an empty swift testing test run output being printed to console.

Prevent `xctest` from running the swift-testing binary by setting the `SWIFT_TESTING_ENABLED` environment variable to 0.

### CODE REVIEW

File: src/debugger/buildConfig.ts [hunk 1]
```diff
@@ -191,28 +186,15 @@ export class TestingDebugConfigurationFactory {
...
                             ...this.baseConfig,
                             program: path.join(xcTestPath, "xctest"),
-                            args: this.addXCTestExutableTestsToArgs([
-                                path.join(
-                                    buildDirectory,
-                                    "debug",
-                                    this.ctx.swiftPackage.name + "PackageTests.xctest"
-                                ),
-                            ]),
+                            args: this.addXCTestExutableTestsToArgs([this.xcTestOutputPath]),
```
Comment (adam-fowler) [inline src/debugger/buildConfig.ts:193]: "addXCTestExutableTestsToArgs" [sic]

**Review by adam-fowler — APPROVED** [review]
I minor typo inherited from another PR

File: src/debugger/buildConfig.ts (hunk 1 above)
Comment (plemarquand) [reply src/debugger/buildConfig.ts:193]: Fixed in #866

**Comment by award999** [comment]
Verified

**Pull request merged**